cas-server-support-jdbc/src/main/java/org/jasig/cas/adaptors/jdbc/QueryDatabaseAuthenticationHandler.java

```diff
@@ -44,10 +44,10 @@ protected final HandlerResult authenticateUsernamePasswordInternal(final Usernam
         }
 
         final String username = credential.getUsername();
-        final String encryptedPassword = this.getPasswordEncoder().encode(credential.getPassword());
+        final String password = credential.getPassword();
         try {
             final String dbPassword = getJdbcTemplate().queryForObject(this.sql, String.class, username);
-            if (!dbPassword.equals(encryptedPassword)) {
+            if (!getPasswordEncoder().matches(password, dbPassword)) {
```

Comment:
Refer to https://github.com/joemccall86/cas/commit/711dced9f7eb1888e4145ebc3c3600bfb59b0cd7

I found that it only made sense to update `QueryDatabaseAuthenticationHandler` and `AcceptUsersAuthenticationHandler` to use the `.matches()` method. Other handlers either used their own encoding (`QueryAndEncodeDatabaseAuthenticationHandler`) or passed the username/password to another system to do the checking (e.g., ldap, jaas, shiro).

I did just realize I missed `FileAuthenticationHandler.java`. I'll go ahead and update that one as well.